package NDnnProto;


message TStochasticConst {
    optional double Prob = 1;
    optional double InhibitoryNeuronProb = 2;
}

